### ISSUE TITLE
[BUG] Can't find information from .env file

### ISSUE THREAD

**Issue by rowanco** [siddarth-patil/DaftRentalBot#5]
**Describe the bug**
As soon as program runs it can't parse the information from the .env file. 

**To Reproduce**
1. Run Bot
2. See error 
Python-dotenv could not parse statement starting at line 21
Python-dotenv could not parse statement starting at line 22
Python-dotenv could not parse statement starting at line 23
Python-dotenv could not parse statement starting at line 24
Python-dotenv could not parse statement starting at line 25
Python-dotenv could not parse statement starting at line 26
Python-dotenv could not parse statement starting at line 21
Python-dotenv could not parse statement starting at line 22
Python-dotenv could not parse statement starting at line 23
Python-dotenv could not parse statement starting at line 24
Python-dotenv could not parse statement starting at line 25
Python-dotenv could not parse statement starting at line 26

When browser is loaded, sample email and password is entered instead of changed ones in .env file.

<img width="547" alt="Screenshot 2023-03-01 at 21 23 18" src="https://user-images.githubusercontent.com/6781912/222268056-9c181513-bc4a-459d-8b77-be6c4696775a.png">

**Comment by siddarth-patil** [comment]
Hi @rowanco thank you for raising the issue. If the sample email and password are being entered then it looks like the .env file was not updated properly. Can you please ensure the following before trying to run it again:
1. Replace existing sample email id and sample password from line 2 and line 3 with your values. 
2. Please make sure that the values for variables `secretUser` and `secretPassword` in .env file are strings.
3. Please make sure to save the .env file before running the script.

If the issue still persist then please show how the .env file looks like by hiding your email and password. I hope this will help you.

**Comment by rowanco** [comment]
I've figured out the issue I was having. I hadn't realised that the daftrentalbot was saved elsewhere and not in my downloads folder. I was editing the .env file in that version as opposed to the version that terminal was actually running. I have since got it running inputting my username and password. Thanks so much for writing this script and your help.

Just to be clear, this only searches properties for rent and not rooms to rent, correct?
